lib/feature/profile/manage_seeds_accounts/widgets/export_seed_sheet/export_seed_save_phrase.dart

```diff
@@ -0,0 +1,106 @@
+import 'package:app/app/service/service.dart';
+import 'package:app/di/di.dart';
+import 'package:app/generated/generated.dart';
+import 'package:collection/collection.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:ui_components_lib/ui_components_lib.dart';
+
+/// Helper method that displays [ExportSeedSavePhrase] sheet.
+ModalRoute<void> exportSeedSavePhraseRoute(List<String> phrase) {
+  return commonBottomSheetRoute(
+    useAppBackgroundColor: true,
+    title: LocaleKeys.saveSeedPhrase.tr(),
+    body: (_, controller) => ExportSeedSavePhrase(
+      phrase: phrase,
+      controller: controller,
+    ),
+  );
+}
+
+/// Widget that shows the seed phrase that could be copied.
+class ExportSeedSavePhrase extends StatelessWidget {
+  const ExportSeedSavePhrase({
+    required this.phrase,
+    required this.controller,
+    super.key,
+  });
+
+  final List<String> phrase;
+  final ScrollController controller;
+
+  @override
+  Widget build(BuildContext context) {
+    return SeparatedColumn(
+      mainAxisSize: MainAxisSize.min,
+      separatorSize: DimensSize.d24,
+      children: [
+        Flexible(
+          child: SingleChildScrollView(
+            controller: controller,
+            child: _wordsField(),
+          ),
+        ),
+        _copyButton(),
+      ],
+    );
+  }
+
+  Widget _textPair(String word, int index) {
+    // ignore: no-magic-number
+    final indexText = index < 10 ? '0$index' : '$index';
```

Comment:
I'm not sure, but is it better to add intl package to the app again and use NumberFormat('00').format(index) instead of this construction?

Reply:
-